service/lib/ops/policyOps.js

```diff
@@ -39,16 +39,26 @@ function insertPolicies (client, policies, cb) {
   client.query(sql, cb)
 }
 
-function deletePolicies (client, ids, cb) {
-  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deletePolicies (client, ids, orgId, cb) {
+  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids}) AND org_id=${orgId}`, utils.boomErrorWrapper(cb))
```

Comment:
We usually are returning 400. In a previous PR  we introduced functions like [`checkUsersOrg `](https://github.com/nearform/labs-authorization/pull/248/files#diff-c675678714399406311ee137b861e90fR216). I don't remember if we also have something similar for policies somewhere.

> since orgs are completely secluded, how likely is it that for this scenario to really happen ?

I think the problem here could be: I'm deleting with the wrong data and I'm getting a 200 if we do not check if what I want to delete has been deleted correctly. **Note**: it may be that we run the check somewhere else, and it would be fine. I was just asking if you knew we checked those values somewhere else.